extensions/ui/org.eclipse.smarthome.ui.paper/gulpfile.js

```diff
@@ -63,6 +64,7 @@ var paths = {
         './node_modules/masonry-layout/dist/masonry.pkgd.min.js',
         './node_modules/sprintf-js/dist/sprintf.min.js',
         './node_modules/bootstrap/dist/js/bootstrap.min.js',
+        './node_modules/angular-material-expansion-panel/dist/md-expansion-panel.min.js',
```

Comment:
Why do you need this here if you have already added the file to angular-bundle? Adding it to one place should be sufficient.